Anlab.Mvc/Controllers/OrderController.cs

```diff
@@ -34,7 +37,28 @@ public OrderController(ApplicationDbContext context, IOrderService orderService,
 
         public IActionResult Create()
         {
-            var model = new OrderEditModel { TestItems = _context.TestItems.AsNoTracking().ToArray() };
+            var prices = _itemPriceService.GetPrices();
+            var items = _context.TestItems.AsNoTracking().ToList();
+
+            var joined = (from i in items
+                join p in prices on i.Code equals p.Code
+                select new TestItem{Analysis = i.Analysis,

```

Comment:
@jSylvestre New model here, TestItemModel or similar, that has these additional calculatable fields.